packages/icons/src/library/columns.js

```diff
@@ -5,7 +5,11 @@ import { Path, SVG } from '@wordpress/primitives';
 
 const columns = (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M19 6H6c-1.1 0-2 .9-2 2v9c0 1.1.9 2 2 2h13c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2zm-4.1 1.5v10H10v-10h4.9zM5.5 17V8c0-.3.2-.5.5-.5h2.5v10H6c-.3 0-.5-.2-.5-.5zm14 0c0 .3-.2.5-.5.5h-2.6v-10H19c.3 0 .5.2.5.5v9z" />
+		<Path
+			fillRule="evenodd"
```

Comment:
Same with these, maybe they work without?

Reply:
This is what happens when I remove 🤷‍♂️.

![CleanShot 2023-11-27 at 17 45 11](https://github.com/WordPress/gutenberg/assets/1813435/9487815d-c1c3-47a1-bf33-e4110e7cf728)

Reply:
Thanks for looking!